kolibri/core/deviceadmin/management/commands/dbrestore.py

```diff
@@ -92,19 +92,19 @@ def select_backup(self, dumps_root):
         if not backups:
             raise RuntimeError("Could not find a database backup}")
```

Comment:
This wasn't part of your PR, but hitting this `RuntimeError` is a poor experience for a non-exceptional scenario. Perhaps just log the message and exit?

Reply:
Seems reasonable.